doc/releasenotes/13_0_0_summary.md

```diff
@@ -60,6 +60,17 @@ For example:
 vtctlclient OnlineDDL commerce complete d08ffe6b_51c9_11ec_9cf2_0a43f95f28a3
 ```
 
+### vtctl/vtctlclient ApplySchema -uuid
```

Comment:
Intuitively `uuid` seems to indicate that it is a single uuid versus a list or a set. `uuids` feels awkward, but maybe `uuid_list`??